torch/nn/attention/flex_attention.py

```diff
@@ -1083,6 +1083,8 @@ def _apply_kernel_options(
     kernel_options.setdefault("PRESCALE_QK", False)
     kernel_options.setdefault("ROWS_GUARANTEED_SAFE", False)
```

Comment:
So I want this to be an ENUM but I think we dont want to provide any guarantees right now on the API so I will leave in this gross state. I plan to add some more sturcture to how we set options since its kinda the wild west right now